client/src/partials/references/references.html

```diff
@@ -1,195 +1,259 @@
-              <th>{{ "COLUMNS.IS_REPORT" | translate}}</th>
-              <th>{{ "COLUMNS.REF_GROUP" | translate}}</th>
-              <th>{{ "COLUMNS.RESULT_ACCOUNT_SCT" | translate}}</th>
-              <th colspan="4">{{ "COLUMNS.ACTIONS" | translate }}</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr ng-repeat="reference in references.data | orderBy:'text'">
-              <td>{{$index + 1}}</td>
-              <td>{{ reference.ref }}</td>
-              <td>{{ reference.text }}</td>
-              <td>{{ reference.position }}</td>
-              <td><span class="glyphicon" ng-class="{'glyphicon-ok' : !!reference.is_report}"></td>
-              <td>{{ reference_groups.get(reference.reference_group_id).text }}</td>
-              <td>{{ section_resultat.get(reference.section_resultat_id).text }}</td>
-              <td><a class="action" ng-click="edit(reference)"><span class="glyphicon glyphicon-pencil"></span></a></td>
-              <td><a class="action danger" ng-click="delete(reference)"><span class="glyphicon glyphicon-trash"></span></a></td>
-            </tr>
-            <tr ng-if="!references.data.length">
-              <td colspan="8">{{ "REFERENCE.NO_REFERENCES" | translate }}</td>
-            </tr>
-          </tbody>
-        </table>
-      </div>
-    </div>
+<div class="flex-content">
+  <div class="container-fluid">
+    <div class="row">
+      <!-- Complete list of registred Reference -->
+      <div class="col-xs-6">
+        <div class="panel panel-default" style="overflow: auto; max-height: 500px;">
+          <div class="panel-heading">
+            {{ "REFERENCE.REGISTERED" | translate }}
+          </div>
 
-    <div class="col-xs-5" ng-switch="session.action">
-      <div class="alert alert-info" ng-switch-default>
-        <h4>{{ "REFERENCE.TITLE" | translate }}</h4>
-        <p>{{ "REFERENCE.ALERT" | translate }}</p>
+          <table class="table table-condensed">
+            <thead>
+              <tr>
+              <th>{{ "TABLE.COLUMNS.NR" | translate}}</th>
+              <th>{{ "TABLE.COLUMNS.REF" | translate}}</th>
+              <th>{{ "TABLE.COLUMNS.LABEL" | translate}}</th>
+              <th>{{ "TABLE.COLUMNS.POSITION" | translate}}</th>
+              <th>{{ "TABLE.COLUMNS.REF_GROUP" | translate}}</th>
+              <th>{{ "TABLE.COLUMNS.RESULT_ACCOUNT_SCT" | translate}}</th>
+              <th colspan="2">{{ "TABLE.COLUMNS.ACTION" | translate }}</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-if="ReferenceCtrl.session.loading" class="text-center">
+                <td colspan="8"><loading-indicator></loading-indicator></td>
+              </tr>
+              <tr 
+              ng-repeat="reference in ReferenceCtrl.references | orderBy:'number' | orderBy:'text' " 
+              ng-class="{'rowSelected' : ReferenceCtrl.reference.id === reference.id}" >
+                <td>{{$index + 1}}</td>
```

Comment:
Is `reference.id` a number?  What is wrong with just using that?

In the case of debtor groups, the id is a UUID which we cannot display.  Using `$index` _might_ be okay, since it helps users easily find things on the page.  The question is, how difficult is it to find a debtor group in a large list of debtor groups without using `$index`?  Would using `class="table-striped"` help?

I'm not sure what the solution is.  This is an acceptable user experience for now - presumably users are not creating debtor groups very often.